neural_lam/utils.py

```diff
@@ -250,7 +251,8 @@ def fractional_plot_bundle(fraction):
     Get the tueplots bundle, but with figure width as a fraction of
     the page width.
     """
-    bundle = bundles.neurips2023(usetex=True, family="serif")
+    usetex = True if shutil.which("latex") else False
```

Comment:
this seems resonable, are you ok with adding this @joeloskarsson?

Reply:
Colleagues had issues with usetex=True on some machines, I appreciate this addition.

Reply:
I also struggle with this when sometimes working on machines without a tex installation. I think this is a good solution. If you don't want to use tex you can use something like ² in units and I think that will look ok. Let's leave further improvements to a more robust rework of the visualization code.

Reply:
Ok, maybe in the case add a comment @SimonKamuk that with this change plots might render incorrectly (but at least not cause an exception to be raised?)

Reply:
Alternatively we just change the units to use ² , together with this change? I think that should look ok both with and without tex (but you might want to double-check that).